app/pb_kits/playbook/pb_caption/_caption.html.erb

```diff
@@ -1,5 +1,7 @@
-<%= content_tag(object.tag, object.text,
-    id: object.id,
-    data: object.data,
-    class: object.classname,
-    aria: object.aria) %>
+<%= content_tag(object.tag,
+  aria: object.aria,
+  id: object.id,
+  data: object.data,
+  class: object.classname) do %>
+  <%= object.children.present? ? capture(&object.children) : object.text %>
+<% end %>
```

Comment:
I modeled this after the [body kit](https://github.com/powerhome/playbook/blob/v6.9.0/app/pb_kits/playbook/pb_body/_body.html.erb)